paasta_tools/utils.py

```diff
@@ -350,8 +350,11 @@ def get_service(self) -> str:
     def job_id(self) -> str:
         return self._job_id
 
-    def get_docker_registry(self) -> str:
-        return get_service_docker_registry(self.service, self.soa_dir)
+    def get_push_docker_registries(self) -> List[str]:
+        return get_service_push_docker_registries(self.service, self.soa_dir)
+
+    def get_primary_docker_registry(self) -> str:
```

Comment:
do you need to rename this function from get_docker_registry?

Reply:
I renamed it to get_primary_docker_registry for it to better represent its purpose.